py/kubeflow/testing/argo_client.py

```diff
@@ -39,6 +39,19 @@ def log_status(workflow):
 @retry(wait_exponential_multiplier=1000, wait_exponential_max=10000,
```

Comment:
I think we should reduce the max_delay to 5 minutes. 20 minutes made sense when we were waiting for the workflow to complete; now we just need to cap it long enough for it to succeed.

Reply:
Done.